shell/app/node_main.cc

```diff
@@ -108,6 +108,10 @@ int NodeMain(int argc, char* argv[]) {
     crash_reporter::CrashReporterWin::SetUnhandledExceptionFilter();
 #endif
 
+    // We do not want to double-set the error level and promise rejection
+    // callback.
+    node::g_standalone_mode = false;
```

Comment:
One more style nit, can you also add `node::g_upstream_node_mode=true` here  and remove the following from the patch

```
diff --git a/src/node.cc b/src/node.cc
index 7dda6b39c3..c084a44544 100644
--- a/src/node.cc
+++ b/src/node.cc
@@ -129,9 +129,6 @@ using v8::Undefined;
 using v8::V8;
 using v8::Value;
 
-bool g_standalone_mode = true;
-bool g_upstream_node_mode = true;
-
 namespace per_process {
 
 // node_revert.h
```

Its better and clear the extern variables are defined either in `node_main` or in electron through `node_bindings`.